src/CMakeLists.txt

```diff
@@ -341,6 +341,9 @@ add_dependencies(tarantool build_bundled_libs preprocess_exports)
 # Re-link if exports changed
 set_target_properties(tarantool PROPERTIES LINK_DEPENDS ${exports_file})
 
+# TODO: Autogenerate the list of internal symbols.
```

Comment:
Nit: I recommend to create a ticket for this TODO and refer to it in the comment so that we don't forget to fix it:

```
# TODO(gh-XXXX): Autogenerate the list of internal symbols.
```